### DISCUSSION TITLE
disabilitare un  automazione .... non manualmente

### DISCUSSION THREAD

**Discussion by Ector73** [indomus/forum#2617] [HUB: Home Assistant]
Ciao a tutti,
ho un automazione che mi attiva l'allarme appena mi allontano da casa, tramite geolocalizzazione del cellulare.
Volevo creare un altra automazione/script che mi "disabiliti" la precedente, se ad esempio ho la baby-sitter in casa e io mi devo allontanare.
Fino ad oggi disabilito manualmente, ma vorrei creare un pulsante virtuale sulla "panoramica" di HA, o sfruttare uno dei pulsanti del "Moes Tuya" per disabilitare l'automazione dell'allarme, ancora meglio sarebbe per un tot di tempo.
In altre parole mi serve un automazione/script che mi disabiliti un altra automazione (per un tempo x)...è possibile?
Ci sto smanettando da un pò, ma non ne vengo fuori.
Grazie

**Comment by snakuzzo** [comment]
Sì, usi il servizio:

```
automation.turn_off
```

**Reply by Ector73** [reply]
Grazie, era sotto gli occhi e non lo vedevo! 🤦
